src/iot/index.spec.js

```diff
@@ -52,5 +53,9 @@ describe('Iot', function() {
     it('appends an instance of Outputs to the class instance', function() {
       expect(iot.outputs).to.be.an.instanceof(Outputs);
     });
+
+    it('appends the supplied FeedTypes module to the class instance', function() {
```

Comment:
I added this test here to be consistent, but I don't see the value. We're testing a variable assignment instead of a behavior, which in my opinion is the same as testing a constant equals a value.